vocabularies/UI.xml

```diff
@@ -1389,6 +1390,59 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Visualization of a note attached to an entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>Administrative data is given by the annotations
+          [`Common.CreatedBy`](Common.md#CreatedBy),
+          [`Common.CreatedAt`](Common.md#CreatedAt),
+          [`Common.ChangedBy`](Common.md#ChangedBy),
+          [`Common.ChangedAt`](Common.md#ChangedAt)
+          on the same entity type.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="NoteType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Title" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Title of the note" />
+          <Annotation Term="Core.LongDescription">
+            <String>The title of a note is hidden with an annotation `@UI.Note/Title/@UI.Hidden`, not with
+            an annotation on the property targeted by `@UI.Note/Title`.</String>
+          </Annotation>
+          <Annotation Term="Validation.ApplicableTerms">
+            <Collection>
+              <String>UI.Hidden</String>
+            </Collection>
+          </Annotation>
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+          <Annotation Term="Core.LongDescription">
+            <String>The property targeted by `@UI.Note/Content` must be annotated with `Core.MediaType` and
+            `Common.SAPObjectNodeTypeReference`. When it is tagged with `Core.IsLanguageDependent`,
```

Comment:
I feel that this requirement is to hard - why can I use notes only in combination with a "SAPObjectNodeType" (btw. it would be good to link a reference here, what this term means and where I can find the list of these types). When I checked the Common vocabulary I recognized that the SAPObjectNode is a complex type with only one property - by intent? (but that's off-topic)

Reply:
`Common.SAPObjectNodeTypeReference` _is_ linked under "Can be annotated with", see the markdown file.

@thomas-kaltenbach, please answer whether a SAP object node type reference is mandatory when working with notes.

Reply:
Maybe a misunderstanding - the link to the other term is good - but step into the shoes of a CAP app developer, who wants to implement a notes reuse service - what do you expect form him (and the description there "The name of the SAP Object Node Type" will not be very helpful for him).

Reply:
I have changed the description of `SAPObjectNodeTypeReference`. But the list of these types is only available SAP-internally, so it makes no sense to document it in a public GitHub repo.

Reply:
> @thomas-kaltenbach, please answer whether a SAP object node type reference is mandatory when working with notes.

For an generic note reuse service it is needed to know which object node type is using the component. I guess this is not needed if a application is defining their own notes and reusing the frontend. But for a generic backend service we do some additional http calls with the application objectID and the sapObjectNodeType for the language/notetype combobox and for the create menu button to display them correct.

Reply:
Thanks, @thomas-kaltenbach! For me this means. that the annotation with "Common.SAPObjectNodeTypeReference" should not be a MUST.

Reply:
For me it is currently difficult to say exactly because we still have to carry out the implementation. But it may well not be a MUST.
As soon as I know more details, I can also open a PR with an adjustment if that is desired.